1.MLP_Binary_Classifier/MLP_utils/parameters.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
+
+import torch
+
+
+@dataclass
+class Parameters:
+    """Class for keeping constants"""
+
+    def __init__(self):
+        # Constants
+        # Subset option yes or no? if no SUBSET_NUMBER won't be used
+        self.SUBSET_OPTION = True
```

Comment:
Is there any way to be more explicit with this variable name? what are you subsetting?

Also, is there a way to ever update this? I don't see this option in the `__init__()`

Reply:
Changes made for updating options via updating defaults. Defaults are updated now via config file